README.md

```diff
@@ -1,22 +1,107 @@
+* Styling: [JSS](https://cssinjs.org/?v=v10.0.3) + [Material UI](https://material-ui.com/)
+* State management: [Redux](https://redux.js.org/) + [Redux-Saga](https://redux-saga.js.org/) + [Reselect](https://github.com/reduxjs/reselect)
+* Template generation: [Plop](https://plopjs.com/) + [Handlebars.js](https://handlebarsjs.com/)
+* Compiling: [Webpack](https://webpack.js.org/) + [Babel](https://babeljs.io/)
+* Forms & Validation: [Formik](https://jaredpalmer.com/formik) + [Yup](https://github.com/jquense/yup)
+* Notifications: [Toastify](https://fkhadra.github.io/react-toastify/)
+
+### Structure
+
+The boiler is set up as a [Yarn](https://yarnpkg.com/) workspace/monorepo, this allows for adding additional workspaces like `Blockchain` or `Server` if required and executing parallel scripts across all spaces.
+
+### Webapp
+
+The web app is structured as a standard React app, the important areas to note is the `/api`, `/domains`, `/containers` & `/components`.
+
+The api interactions have helpers to automatically format form field data, collect access token headers and allow for reducer-esque api calling functions.
+
+The domains folder acts as the apps main Daemon & singleton business logic management components, general app actions are managed here, reducers for the domain state, selectors etc can be found here. Domains are meant to be globally accessable services facilitated through Redux-Saga for async actions
+
+The containers folder allows for managing the business logic of constructing selectors, action dispatch functions, and any complex operations that should be managed seperately from the mark up.
+
+The components folder is for markup files & styling.
+
+## Install
+
+The goal of our local build process is to abstract away the complexity with smart contracts so that you can focus on adding useful new functionality through React-based bounties that are easy to get started on.
+
+### Prerequisites
+
+1. [Node v10](https://github.com/nvm-sh/nvm) or higher.
+2. [Yarn](https://yarnpkg.com/).
+3. [mongodb](https://www.mongodb.com/).
+
+First, run yarn to install the workspace dependancies:
 
 ```
-yarn run build:dev
+yarn install
 ```
-or
+
+On Linux, setting up `mongodb` is as easy as `sudo apt install -y mongodb`, which will also start it automatically. You can stop/restart your local DB any time with `sudo systemctl stop mongodb`, or get its status with `sudo systemctl status mongodb`. I recommend using the simple [robo3t](https://robomongo.org/download) to view and edit your DB easily (you'll need to set DApps to `APPROVED` to see `Edit` and `Withdraw` options and to see them appear in Categories etc.)
+
+### Mongo set up
+
+1. `export DB_CONNECTION=mongodb://localhost:27017/mydb`. Make sure you have `DB_CONNECTION` set as an ENV variable so the app knows where to find your local DB.
+2. TODO`yarn run build:localhost`. This will:
```

Comment:
What does the `TODO` mean? Also, it's `yarn run build:local`.

Reply:
Probably a stray string, will remove